apps/frontend/src/index.tsx

```diff
@@ -12,6 +12,8 @@ import { store } from "~/libs/modules/store/store.js";
 import { Auth } from "~/pages/auth/auth.jsx";
 import { NotFound } from "~/pages/not-found/not-found.jsx";
 
+import { Landing } from "./libs/components/landing/libs/components/hero-section/hero-section.js";
```

Comment:
You need to have a landing component 
```suggestion
import { Landing } from "./libs/components/landing/landing.jsx";
```

and this landing component, will combine multiple sections on it as sub components